ci-operator/step-registry/openstack-k8s-operators/deploy/openstack-k8s-operators-deploy-commands.sh

```diff
@@ -91,7 +91,7 @@ if [[ "$SERVICE_NAME" == "OPENSTACK" ]]; then
   export ${SERVICE_NAME}_CR=/go/src/github.com/${DEFAULT_ORG}/${OPENSTACK_OPERATOR}/config/samples/core_v1beta1_openstackcontrolplane.yaml
 fi
 
-make ceph TIMEOUT=90
+make ceph DATA_SIZE=2Gi TIMEOUT=90
```

Comment:
@viroel @Sandeepyadav93 not sure if you have different thoughts on the value we want to pass here.